README.md

```diff
@@ -120,6 +120,17 @@ This behavior is odd. After hitting `Enter` after the first line, nothing happen
 
 If you try to push again, you should be prompted for a new username and password and those will be stored in your keychain for future use.
 
+## Build Logs
```

Comment:
@j-brammer Want to add this to the table of contents?

Reply:
@lukemoderwell Done